File: TVTComment/Model/IPC/IPCMessage/CloseIPCMessage.cs
using System.Collections.Generic;

namespace TVTComment.Model.IPC.IPCMessage
{
    class CloseIPCMessage : IIPCMessage
    {
        public string MessageName => "Close";
        public IEnumerable<string> Encode()
        {
            return System.Array.Empty<string>();
        }
        public void Decode(IEnumerable<string> content)
        {
        }
    }
}

File: Viewer/TVTComment/IPC/IPCMessage/CloseIPCMessage.cpp
#include "stdafx.h"
#include "CloseIPCMessage.h"

namespace TVTComment
{
	std::string CloseIPCMessage::GetMessageName() const
	{
		return u8"Close";
	}
	std::vector<std::string> CloseIPCMessage::Encode() const
	{
		return{};
	}
	void CloseIPCMessage::Decode(const std::vector<std::string> &)
	{
	}
}